.github/workflows/notebook-controller-images-updater.yaml

```diff
@@ -68,12 +97,12 @@ jobs:
     - name: Update related files
       id: apply-changes
       run: |
```

Comment:
Logic of this complexity could be warranted to pull out into a separate script for ease of maintenance (just a suggestion!)